---
layout: home
category: home
---

This website hosts data related to gene regulatory networks built using published maize RNA-Seq datasets.

- Regulation of gene expression is central to many biological processes.  Gene regulatory networks (GRNs) link transcription factors to their target genes and represent a map of potential transcriptional regulation.  
- While there have been notable successes in applying co-expression based GRNs to identify important regulatory networks in plant species there remains a substantial gap in our knowledge of how to develop GRNs from large scale transcriptome datasets that are now available for optimal use in crop improvement.  -
- Using publicly available maize transcriptome datasets we constructed 44 GRNs that represent potential regulatory relationships between TFs and other genes in different populations of samples (cross-tissue, cross-genotype, tissue-and-genotype, etc).  
- Through examining the level of functional enrichment in different types of networks, we explored ways to extract true positive interactions while controlling for false positives.  In order to assess the performance of GRNs in predicting functional TF/target regulatory relationships we used natural variation datasets including a total of 42 paired B73 - Mo17/Oh43/PH207/B37 tissues to validate network predictions.  
- Overlaying predicted TF-target interactions with previous eQTL mapping results provided support for 68 transcription factors being the source underlying 74 previously identified trans-eQTL hotspots.  
- As a proof of concept, we confirmed previous findings that the R1 gene regulates the anthocyanin biosynthesis pathway using our method, and demonstrated that the CONSTANS-LIKE transcription factor col11 and the bHLH transcription factor myc7 might act as crucial regulators for the photosynthesis light reaction pathway and jasmonic acid biosynthesis pathway, respectively.


