---
layout: home
category: home
---

<img src="/maizeGRN/images/maizeGRN.png" height="250px">

**MaizeGRN** hosts data related to gene regulatory networks built using published maize RNA-Seq datasets.

- Regulation of gene expression is central to many biological processes.  Gene regulatory networks (GRNs) link transcription factors (TFs) to their target genes and represent a map of potential transcriptional regulation.  
- While there have been notable successes in applying co-expression based GRNs to identify important regulatory networks in plant species, there remains a substantial gap in our knowledge of how to develop GRNs from large scale transcriptome datasets for optimal use in crop improvement.
- By consistently analyzing a large number of public maize transcriptome datasets including >6000 RNAseq samples, we generated 44 co-expression based GRNs that represent potential regulatory relationships between TFs and other genes in different populations of samples (cross-tissue, cross-genotype, tissue-and-genotype, etc).
- As a proof of concept, integrating information from our TF-target predictions and previous eQTL mapping results provided support for 68 TFs underlying 74 previously identified trans-eQTL hotspots. These putative trans regulators span a variety of metabolic pathways, including a well-studied R1 regulating the anthocyanin biosynthesis pathway, as well as a less studied CONSTANS-LIKE transcription factor COL11 and a bHLH transcription factor MYC7 as crucial regulators for the photosynthesis light reaction pathway and jasmonic acid biosynthesis pathway, respectively.
- This study highlights the utility of a large set of co-expression based GRNs in understanding the landscape of maize transcriptional regulation, elucidating important plant pathways and providing potential targets for breeding or biotechnology applications. 
- [1,824 TF-target interactions supported by at least two GRNs and two previous eQTL studies](#)
